packages/core/src/components/Modal/layouts/ModalSideBySideLayout/__stories__/ModalSideBySideLayout.stories.tsx

```diff
@@ -179,6 +179,7 @@ export const Wizard: Story = {
               <FieldLabel labelText="Language preferences" />
               <Dropdown
                 insideOverflowWithTransformContainer
+                insideLayerContext
```

Comment:
I'd also maybe rename the prop

Reply:
I couldn't think of a better name 🤷🏼 @rivka-ungar wdyt?